### PR TITLE
Compatibility with Rails 5.0

### PR DESCRIPTION
**Pull request by pschambacher**
@blowmage 
Travis doesn't seem very happy ... I've been having difficulties using ruby versions like `2.1` lately and had to explicitly specify `2.1.8`...

### CODE REVIEW

File: test/rails/action_controller/test_assertions.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_assert_template
   def test_routing_assertions
     params = { :controller => "models", :action => "index" }
     path = "/models"
-    assert_generates path, params
+    assert_generates path, params.clone
```
Comment (pschambacher) [inline test/rails/action_controller/test_assertions.rb]: This might be remove if [this patch](https://github.com/rails/rails/pull/23396) gets merged.

**Comment by pschambacher** [comment]
@blowmage I think I fixed the build. Also I removed the `clone` methods now that my patch was merged. Sadly it didn't get shipped with 5.0.0.beta2... so it will have to wait for beta3/rc1.
Good to go? :smile:

**Comment by pschambacher** [comment]
@blowmage ?

**Comment by pschambacher** [comment]
Hello @blowmage would you mind having a look at this Pull Request? Let me know if something needs to be changed.

**Comment by timhaines** [comment]
Oh hey, thanks for putting this together @pschambacher!

**Comment by pschambacher** [comment]
Hi @blowmage, any update here?

**Comment by glaszig** [comment]
with this branch and `require 'minitest/rails` in my `test_helper.rb`, tests are run twice.

```
$ bin/rails test test/controllers
Running via Spring preloader in process 80906
Run options: --seed 52424

# Running:

.........................................................

Finished in 8.625397s, 6.6084 runs/s, 16.3471 assertions/s.

57 runs, 141 assertions, 0 failures, 0 errors, 0 skips
Run options: --seed 654

# Running:

.........................................................

Finished in 4.320855s, 13.1918 runs/s, 32.6324 assertions/s.

57 runs, 141 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by Mehonoshin** [comment]
Any news here, folks?

**Comment by pschambacher** [comment]
Just waiting for @blowmage to review this PR

**Comment by Mehonoshin** [comment]
@blowmage :heart:

**Comment by blowmage** [comment]
Now that Rails 5 is getting closer to a release I'm circling back to minitest-rails. Unfortunately I can't accept this PR as-is because it goes against my plans for how to support different versions of Rails. My plan is to not support Rails 5 and Rails 4.1/4.2 on the same release. The 2.x gems will support Rails 4.2/4.2 on the [2.x branch](https://github.com/blowmage/minitest-rails/tree/2.x), similar to how the 1.x gems support Rails 3.0-4.0 on the [1.x branch](https://github.com/blowmage/minitest-rails/tree/1.x).

Master was just switched to Rails 5. Can you take a look at master and rebase this PR with what might be missing?

**Comment by blowmage** [comment]
Actually, if you do find something that is missing from the master branch please open a new PR. Gonna close this one. Thanks!

**Pull request closed without merging**